src/client/bb-client.js

```diff
@@ -262,6 +262,17 @@ function validTokenResponse() {
   return false;
 }
 
+function isFakeOAuthToken(){
+  if (validTokenResponse()) {
+    var token = getPreviousToken();
+    if (token && token.state) {
+      var state = JSON.parse(sessionStorage[token.state]);
```

Comment:
Doesn't this assume `BBClient.settings.fullSessionStorageSupport = false`? What about the case where it's `true`?

Reply:
I think this should work for that flag when set to `true` or `false`, please let me know if I'm missing something. In either case, the `fake_token_response` property can only be exposed off the state in sessionStorage, whether run against a secured or open FHIR endpoint. When we get the token back, we check session storage for the state value and check the `fake_token_response` property from that once the JSON is parsed.